.travis.yml

```diff
@@ -0,0 +1,26 @@
+
+language: node_js
+
+node_js:
+ - "stable"
+
+services:
+ - postgresql
+
+before_install:
+ - npm install sequelize-cli -g
+   
+
+before-script:
+ - npm add sequelize-cli
+
+before_script:
```

Comment:
@davidfrank96 You have two before_script sections. Can you combine them into 1?

Reply:
Okay...